### PR TITLE
Bug 2000633: fix observer dashboard variables  dropdown when screen size is reduced

### PR DESCRIPTION
**Pull request by debsmita1**
**Fixes:**
https://issues.redhat.com/browse/ODC-6309

**Solution description:**
Fixed the top margin of the dropdowns

**GIF:**
dev-observing-page

### CODE REVIEW

**Review by vikram-raj — COMMENTED** [review]
/lgtm

File: frontend/public/components/monitoring/dashboards/index.tsx [hunk 1]
```diff
@@ -721,19 +725,19 @@ const MonitoringDashboardsPage: React.FC<MonitoringDashboardsPageProps> = ({ mat
...
       )}
       <div className="co-m-nav-title co-m-nav-title--detail">
-        {!namespace && <HeaderTop />}
-        <div className="monitoring-dashboards__variables">
-          <div
-            className={cx('monitoring-dashboards__dropdowns', {
-              'monitoring-dashboards__variable-dropdowns': namespace,
-            })}
-          >
+        {!namespace && <HeaderTop namespace={namespace} />}
```
Comment (invincibleJai) [inline frontend/public/components/monitoring/dashboards/index.tsx:728]: this doesn't look right, if `namespace` is not there then you are passing as a prop to `HeaderTop`, and `HeaderTop` is rendered only if the namespace is not there based on current logic.

File: frontend/public/components/monitoring/_monitoring.scss [hunk 2]
```diff
@@ -156,6 +156,10 @@ $tooltip-background-color: var(--pf-global--BackgroundColor--dark-100);
   margin-top: -20px;
 }
 
+.monitoring-dashboards__options--top-margin {
+  margin-top: 0px;
```
Comment (invincibleJai) [inline frontend/public/components/monitoring/_monitoring.scss:160]:
```suggestion
  margin-top: 0;
```

File: frontend/public/components/monitoring/dashboards/index.tsx (hunk 1 above)
Comment (debsmita1) [reply frontend/public/components/monitoring/dashboards/index.tsx]: I totally overlooked this, thanks for catching it

**Comment by kyoto** [comment]
I think it would be possible and preferable to fix this with just CSS changes.

The admin perspective puts all the dropdowns inside the `co-m-nav-title` container, so the CSS rules could use that.

In other words, something like
```
.co-m-nav-title .monitoring-dashboards__options {
  margin-top: -20px;
}
```
and
```
.monitoring-dashboards__variables {
  ...
  margin-top: -20px;
}
.co-m-nav-title .monitoring-dashboards__variables {
  margin-top: 0;
}
```

I think this would make more sense since it is really where the dropdowns appear on the page that determines what their top margin should be. Also, it means we won't need to pass `namespace` around.

**Pull request merged**